foundation-models/openai/src/test/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClientGeneratedTest.java

```diff
@@ -244,21 +242,19 @@ void history() {
   void embedding() {
     stubForEmbedding();
 
-    final var result =
-        client.embedding(
-            new EmbeddingsCreateRequest()
-                .input(EmbeddingsCreateRequestInput.create("Hello World")));
+    final var response =
+        client.embedding(new OpenAiEmbeddingRequest("Hello World")).getOriginalResponse();
```

Comment:
Please test the `EmbeddingResponse.getEmbeddings()` here instead of it's own test class